front/src/main/ui/pages/Home/firstPage/FirstPage.module.css

```diff
@@ -0,0 +1,73 @@
+.homeTitle {
+    width: 300px;
+    margin: 0 auto 105px auto;
+    position: relative;
+}
+
+
+
+.homeGreenText {
```

Comment:
dont use class name based on color. It may be changed

Reply:
done